inputfiles/sherpa/LHC-NLO-ZplusJet/Run.dat

```diff
@@ -17,6 +19,8 @@
 (processes){
   Process 93 93 -> 13 -13 93;
   Order (*,2);
+  CKKW sqr(QCUT/E_CMS);
+  Cut_Core 1;
```

Comment:
What does the cut_core mean?

Reply:
It makes sure that the QCUT is only applied on the first jet. So it kind of does the same as a jet selection.